docs/src/kubectl-plugin.md

```diff
@@ -34,7 +34,7 @@ For example, let's install the 1.18.1 release of the plugin, for an Intel based
 64 bit server. First, we download the right `.deb` file.
 
 ``` sh
-$ wget https://github.com/cloudnative-pg/cloudnative-pg/releases/download/v1.18.1/kubectl-cnpg_1.18.1_linux_x86_64.deb
+wget https://github.com/cloudnative-pg/cloudnative-pg/releases/download/v1.18.1/kubectl-cnpg_1.18.1_linux_x86_64.deb
 ```
```

Comment:
Why 1.18?

Reply:
Should this match with the build version?

Reply:
I believe this is just a missing place where your release script should have upgraded the version.

Reply:
yeah for sure, because I only removed the $ because testing the copy&paste failed